.github/workflows/push-to-prod-draft.yml

```diff
@@ -29,8 +29,8 @@ jobs:
           python gen_docker_compose.py rockpi -o docker-compose.yml
           ROCKPI="-rockpi"
           echo "ROCKPI=$ROCKPI" >> $GITHUB_ENV
-      - name: Generate Docker Compose for RaspberryPi
-        if: matrix.sbc == 'raspi'
+      - name: Generate Docker Compose for RaspberryPi & RAK
```

Comment:
This should work, but not sure we want to make this the pattern for all vendors we support. They won't all map neatly to an existing Docker Compose template. Might be more maintainable to add a config for each vendor to gen_docker_compose, settings.ini, etc.

Reply:
Can we make it based on the case? In this one, it fits perfectly fine and adding more code could increase complexity. But if there will be any other one, that wouldn't fit perfectly, then we could create a new one for it. Deal? :smiley: